src/schema.graphql

```diff
@@ -12,6 +12,7 @@ type Mutation {
   report(id: ID!, description: String): Report
   disable(id: ID!): ID
   enable(id: ID!): ID
+  addSocialMediaAccount(url: String!): Boolean!
```

Comment:
It’s not a account using it?

Reply:
what do you mean? change the name to addSocialMedia

Reply:
Yeah for example. I find account on the user node a bit missleading as it might suggest third party logins.

Reply:
Also should we maybe suggest in the name where we are adding it? For example: `addUserSocialMedia`?

Reply:
Yeah